### PR TITLE
Support indexing v2

### PR DESCRIPTION
**Pull request by AndreiBarsan**
Meant to correspond to v2 lidar and camera indexes, which are now live in the bucket.

### CODE REVIEW

**Comment by AndreiBarsan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #51** <a href="https://app.graphite.dev/github/pr/pit30m/pit30m/51" target="_blank">Graphite</a> 
    * **PR #55** <a href="https://app.graphite.dev/github/pr/pit30m/pit30m/55" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/pit30m/pit30m/55?utm_source=stack-comment).

File: pit30m/data/log_reader.py [hunk 1]
```diff
@@ -376,7 +375,8 @@ def utm_poses_dense(self) -> Tuple[np.ndarray, np.ndarray]:
         submaps = [UUID(bytes=submap_uuid_bytes.ljust(16, b"\x00")) for submap_uuid_bytes in mrp["submap_id"]]
 
         try:
-            xyzs = self._map.to_utm(xyzs, submaps)
+            # XXX: check with Julieta, is it OK to keep this strict?
+            xyzs = self._map.to_utm(xyzs, submaps, strict=False)
```
Comment (una-dinosauria) [inline pit30m/data/log_reader.py:379]: I think it's ok to keep this as non-strict for now. Thanks for checking!

**Review by una-dinosauria — APPROVED** [review]
LGTM!

File: pit30m/data/log_reader.py (hunk 1 above)
Comment (AndreiBarsan) [reply pit30m/data/log_reader.py]: Awesome, done and updated the doc to make it more clear what the returns mean.

**Pull request merged**